camp/pom.xml

```
@@ -70,7 +50,7 @@
                 <configuration>
                     <archive>
                         <manifest>
-                            <mainClass>ch.bfh.zodnrk.camp.Application</mainClass>
+                            <mainClass>ch.bfh.zodnrk.camp.DemoApplication</mainClass>
```

Should probably not be called 'DemoApplication'.
```suggestion
                            <mainClass>ch.bfh.zodnrk.camp.Application</mainClass>
```

---

Thanks. Fixed as well.